profiles/config.py

```diff
@@ -0,0 +1,38 @@
+from configparser import RawConfigParser
+from typing import Dict
+
+
+class Config(object):
+    def __init__(self, orcid: Dict[str, str]) -> None:
+        self.orcid = orcid
+
+
+class DevConfig(Config):
+    DEBUG = True
```

Comment:
shouldn't there be a `False` default for these constants in the base `Config`?

Reply:
Might be a bit clearer. Flask has a default of `False` though.